cmd/porter/credentials.go

```diff
@@ -0,0 +1,151 @@
+package main
+
+import (
+	"github.com/deislabs/porter/pkg/porter"
+	"github.com/deislabs/porter/pkg/printer"
+	"github.com/spf13/cobra"
+)
+
+func buildCredentialsCommand(p *porter.Porter) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "credentials",
+		Aliases: []string{"cred"},
+		Short:   "Credentials commands",
+	}
+
+	cmd.AddCommand(buildCredentialsAddCommand(p))
+	cmd.AddCommand(buildCredentialsEditCommand(p))
+	cmd.AddCommand(buildCredentialsGenerateCommand(p))
+	cmd.AddCommand(buildCredentialsListCommand(p))
+	cmd.AddCommand(buildCredentialsRemoveCommand(p))
+	cmd.AddCommand(buildCredentialsShowCommand(p))
+
+	return cmd
+}
+
+func buildCredentialsAddCommand(p *porter.Porter) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "add",
+		Short: "Add Credential",
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return nil
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			p.PrintVersion()
+			return nil
+		},
+	}
+	return cmd
+}
+
+func buildCredentialsEditCommand(p *porter.Porter) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "edit",
+		Short: "Edit Credential",
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return nil
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			p.PrintVersion()
+			return nil
+		},
+	}
+	return cmd
+}
+
+func buildCredentialsGenerateCommand(p *porter.Porter) *cobra.Command {
+	opts := porter.CredentialOptions{}
+	cmd := &cobra.Command{
+		Use:   "generate [name]",
+		Short: "Generate Credential Set",
+		Long: `Generate a CNAB Credential Set.
```

Comment:
No need to call it a CNAB credential set here. Elsewhere in the helptext I've been calling it a named set of credentials if that helps give you a better long set of text.